src/data/zwave_js.ts

```diff
@@ -284,11 +284,13 @@ export const migrateZwave = (
 
 export const fetchZwaveNetworkStatus = (
   hass: HomeAssistant,
-  entry_id: string
+  device_or_entry_id: string,
+  id_type: "entry_id" | "device_id" = "entry_id"
```

Comment:
Can we just make this 2 optional params?
```ts
entry_id?: string,
device_id?: string,
```

```ts
if (!device_id && !entry_id) {
	throw new Error("Either device or entry id should be supplied");
}
```

Reply:
Too make it easier, change the 2 string params to an object that has the 2 string params, so you don't have to pass undefined when you want to use the `device_id` and it helps prevent errors 😄